### ISSUE TITLE
UIButton.isSelected as Observable

### ISSUE THREAD

**Issue by tkallioras** [ReactiveX/RxSwift#1109]
Hello there.

I saw that there has been a similar issue posted, but I run into the following issue.

I want to check the isSelected state of some UIButtons in order to feed the isEnabled state of another button. I wanted the latter button enabled only when one of the other buttons isSelected state is set to true.
However I saw that you have chosen to keep the isSelected property only as an Observer.

How can we handle a situation as described above?

**Comment by kzaher** [comment]
Hi @tkallioras ,

```
let state: Observable<Bool>

state.bindTo(button.rx.isSelected)
    .disposed(by: disposeBag)

state.bindTo(button.rx.isEnabled)
    .disposed(by: disposeBag)
```

**Comment by weechong-ong** [comment]
Sent from my Samsung Galaxy smartphone.
-------- Original message --------From: Krunoslav Zaher <notifications@github.com> Date: 26/02/2017  19:32  (GMT+08:00) To: ReactiveX/RxSwift <RxSwift@noreply.github.com> Cc: Subscribed <subscribed@noreply.github.com> Subject: Re: [ReactiveX/RxSwift] UIButton.isSelected as Observable (#1109) 
Closed #1109.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/ReactiveX/RxSwift","title":"ReactiveX/RxSwift","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/ReactiveX/RxSwift"}},"updates":{"snippets":[{"icon":"DESCRIPTION","message":"Closed #1109."}],"action":{"name":"View Issue","url":"https://github.com/ReactiveX/RxSwift/issues/1109#event-977398686"}}}
